native/templates/openharmony/entry/src/main/ets/MainAbility/workers/cocos_worker.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2021 Huawei Device Co., Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import worker from '@ohos.worker';
+import nativerender from "libcocos2d.so";
```

Comment:
The so name is changed in 3.6, will be cocos.
All codes in template should keep compatibility in future.

Reply:
open a new issue : https://github.com/cocos/3d-tasks/issues/13655